<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MwcToolbar</class>
 <widget class="QWidget" name="MwcToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>157</width>
    <height>702</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>157</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>157</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources_desktop.qrc">:/img/TBLogo@2x.svg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="totalMwc">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>This is the total MWC in all of your accounts, some might be locked or not confirmed</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: Open Sans;
font-style: normal;
font-weight: bold;
font-size: 22px;
line-height: 35px;
/* identical to box height */


color: #FFFFFF;</string>
       </property>
       <property name="text">
        <string>mwc</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QToolButton" name="sendToolButton">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Send MWC</string>
       </property>
       <property name="text">
        <string notr="true">Send</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/img/Send@2x.svg</normaloff>:/img/Send@2x.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="receiveToolButton">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Receive MWC</string>
       </property>
       <property name="text">
        <string notr="true">Receive</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_desktop.qrc">
         <normaloff>:/img/Receive@2x.svg</normaloff>:/img/Receive@2x.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="finalizeToolButton">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Finalize File Transaction</string>
       </property>
       <property name="text">
        <string notr="true">Finalize</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_desktop.qrc">
         <normaloff>:/img/Finalize@2x.svg</normaloff>:/img/Finalize@2x.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="transactionToolButton">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Click here to view your transactions</string>
       </property>
       <property name="text">
        <string notr="true">Transactions</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_desktop.qrc">
         <normaloff>:/img/Transactions@2x.svg</normaloff>:/img/Transactions@2x.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="airdropToolButton">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Click here to claim your MWC from the Airdrop</string>
       </property>
       <property name="text">
        <string notr="true">Airdrop</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/img/Airdrop@2x.svg</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="hodlToolButton">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>62</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Click to register your coins for the HODL program</string>
       </property>
       <property name="text">
        <string notr="true">HODL</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_desktop.qrc">
         <normaloff>:/img/HODL@2x.svg</normaloff>:/img/HODL@2x.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources_desktop.qrc"/>
 </resources>
 <connections/>
</ui>
